Dockerfile

```diff
@@ -139,6 +139,9 @@ ENTRYPOINT ["/app/scripts/run-docker.sh"]
 
 CMD ["migrate-and-serve"]
 
+HEALTHCHECK --interval=2s --timeout=2s --retries=3 --start-period=20s \
+  CMD curl -f http://localhost:8000/health/liveness || exit 1
```

Comment:
@khvn26 I only just now noticed the Wolfi Docker images don't contain curl. Don't merge this until it's corrected, sorry about that!